chevah/compat/tests/elevated/test_capabilities.py

```diff
@@ -77,13 +77,14 @@ def test_getCurrentPrivilegesDescription(self):
                 'SeIncreaseWorkingSetPrivilege:0, SeTimeZonePrivilege:0, '
                 'SeCreateSymbolicLinkPrivilege:0'
                 )
-            if int(self.os_version.split('.')[2]) >= 14393:
-                # Starting with Win 2016 there's an extra default capability.
+            if self.os_version == 'nt-10.0':
+                # On latest Windows there is an extra capability by default.
                 expected_capabilities += (
                     ', SeDelegateSessionUserImpersonatePrivilege:0'
                     )
-            if int(self.os_version.split('.')[2]) >= 17763:
-                # This is Win 2019 or newer, some privileges are different:
+            if 'win-2019' in self.hostname:
+                # On the Windows 2019 slave some privileges are different,
+                # as UAC is disabled:
```

Comment:
I'm not sure this is the reason… Why do you say _UAC is disabled_? I haven't disabled it manually or through Salt. This is how things stand by default in Windows 2019. 

The only thing I've tried was to include `buildslave` user in the `Backup Operators` group, but that doesn't changes privileges, as `buildslave` is part of the `Administrators` group.